tests/virtualization/universal/register_guests.pm

```diff
@@ -16,13 +16,20 @@ use version_utils;
 sub run_test {
     my ($self) = @_;
     # Use serial terminal, unless defined otherwise. The unless will go away once we are certain this is stable
-    $self->select_serial_terminal unless get_var('_VIRT_SERIAL_TERMINAL', 1) == 0;
+    # $self->select_serial_terminal unless get_var('_VIRT_SERIAL_TERMINAL', 1) == 0;
+    select_console 'root-console';
+    my %ltss_products = @{get_var_array("LTSS_PRODUCTS")};
 
     foreach my $guest (keys %virt_autotest::common::guests) {
         record_info "$guest", "Registrating $guest against SMT";
         my ($sles_running_version, $sles_running_sp) = get_os_release("ssh root\@$guest");
+        my $version_id = $sles_running_sp != 0 ? "$sles_running_version.$sles_running_sp" : $sles_running_version;
         if ($sles_running_version >= 12) {
             assert_script_run("ssh root\@$guest SUSEConnect -r " . get_var('SCC_REGCODE') . " -e " . get_var("SCC_EMAIL"));
```

Comment:
@tonyyuan1  This comment is not related to your PR. But do you find this line is registering wrongly too? The SCC_REGCODE is the correct regcode for the product that the host is installed to, rather than all vms which mostly are different from host version. HDYT?

Reply:
I found the values of SCC_REGCODE for different product were identical in OSD, but varied in arch'es. So the SCC_REGCODE from host may work for most guests except LTSS ones. It is just my observation, not sure if it is correct.

Reply:
Yes, Julie answered for me :)

Reply:
Okay, that's fine